content/docs/rfcs/62/README.md

```diff
@@ -0,0 +1,1077 @@
+  int64 contact_request_local_state = 16;
+  int64 contact_request_local_clock = 17;
+  int64 contact_request_remote_state = 18;
+  int64 contact_request_remote_clock = 19;
+  string display_name = 20;
+}
+```
+
+
+#### Payload
+
+| Field | Name | Type | Description |
+| ----- | ---- | ---- | ---- |
+| 1 | last_updated_locally | `uint64` | Timestamp of last local update | 
+| 2 | id | `string` | id of the contact synced |
+| 3 | profile_image | `string` |  `base64` encoded profile picture of the user |
+| 4 | ens_name | `string` | ENS name of the contact |
+| 5 | `array[string]` | Array of `system_tags` for the user, this can currently be: `":contact/added", ":contact/blocked", ":contact/request-received"`|
+| 7 | local_nickname | `string` | Local display name of the contact |
+| 9 | added | `bool` | Wether the contact is added |
+| 10 | blocked | `bool` | Wether the contact is blocked |
+| 11 | muted | `bool` | Wether the contact is muted |
+| 12 | removed | `bool` | Wether the contact is removed |
+| 13 | has_added_us | `bool` | Wether the contact has added us |
+| 14 | verification_status | `int64` | The verification status of the contact |
+| 15 | trust_status | `int64` | The trust status of the contact |
+| 16 | contact_request_local_state | `int64` | The local contact request state |
+| 17 | contact_request_local_clock | `int64` | The local contact request clock |
+| 18 | contact_request_remote_state | `int64` | The remote contact request state |
+| 19 | contact_request_remote_clock | `int64` | The remote contact request clock |
+
+### SyncInstallationPublicChat
+
+The node uses `SyncInstallationPublicChat` message to synchronize in a best-effort the public chats to other devices.
+
+```protobuf
+message SyncInstallationPublicChat {
+  uint64 clock = 1;
+  string id = 2;
+}
+```
+
+#### Payload
+
+| Field | Name | Type | Description |
+| ----- | ---- | ---- | ---- |
+| 1 | clock | `uint64` | clock value of the chat | 
+| 2 | id | `string` | id of the chat synced |
+
+### SyncPairInstallation
+
+The node uses `PairInstallation` messages to propagate information about a device to its paired devices.
+
+```protobuf
+message SyncPairInstallation {
+  uint64 clock = 1;
+  string installation_id = 2;
+  string device_type = 3;
+  string name = 4;
+  // following fields used for local pairing
+  uint32 version = 5;
+}
+```
+
+#### Payload
+
+| Field | Name | Type | Description |
+| ----- | ---- | ---- | ---- |
+| 1 | clock | `uint64` | clock value of the chat | 
+| 2| installation_id | `string` | A randomly generated id that identifies this device |
+| 3 | device_type | `string` | The OS of the device `ios`,`android` or `desktop` |
+| 4 | name | `string` | The self-assigned name of the device |
+
+### ChatIdentity
+
+`ChatIdentity` represents the user defined identity associated with their public chat key.
+
+```protobuf
+message ChatIdentity {
+  uint64 clock = 1;
+  string ens_name = 2;
+  map<string, IdentityImage> images = 3;
```

Comment:
how many images can the `ChatIdentity` house? this will potentially break the message size limitation set by waku